lib/helpers/id-helper.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { customAlphabet } = require('nanoid');
+
+const randomValue = digits => customAlphabet('ABCDEFGHIJKLMNOPQRSTUVWXYZ123456789', digits);
```

Comment:
Es un buen caso este para agregar el memoize, asi evitamos generar nuevos _alphabets_ una y otra vez, con los mismos caracteres y la misma longitud..

```suggestion
const randomValue = memoize(digits => customAlphabet('ABCDEFGHIJKLMNOPQRSTUVWXYZ123456789', digits));
```